.clang-format

```diff
@@ -0,0 +1,48 @@
+# Formatting approximately used in Solidity's C++
+#
+# See https://clang.llvm.org/docs/ClangFormatStyleOptions.html
```

Comment:
I think this comment block is all that is needed. I think it's better to look up the options in the official documentation instead.